stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/call/connection/Subscriber.kt

```diff
@@ -584,6 +604,69 @@ internal class Subscriber(
         }
     }
 
+    override fun onRemoveStream(stream: MediaStream?) {
```

Comment:
Not sure but I think we may not get this callback from webrtc atall .. have to check 

but why do we need this anyway? We already know of the tracks that were removed